simulator/func_sim/rf/rf.h

```diff
@@ -68,24 +68,45 @@ class RF
         return 0u;
     }
 
-    void write( Register num, RegDstUInt val, int8 accumulating_instr = 0)
+    void write( Register num, RegDstUInt val, int8 is_accumulating_instr = 0, int8 loadlr = 0, uint32 mask = 0x0)
```

Comment:
Mask should have the same type as val (RegDstUInt), so Clang can compile that for 128 bits.